src/AspNetCoreModuleV2/InProcessRequestHandler/inprocesshandler.cpp

```diff
@@ -30,18 +30,28 @@ IN_PROCESS_HANDLER::OnExecuteRequestHandler()
 
     if (FAILED(hr))
     {
-        // TODO remove com_error?
-        /*_com_error err(hr);
-        if (ANCMEvents::ANCM_START_APPLICATION_FAIL::IsEnabled(m_pW3Context->GetTraceContext()))
+        if (!m_pApplication->QueryConfig()->QueryDisableStartUpErrorPage())
```

Comment:
Can we return a different IAPPLICATION instance if app failed to start?

Reply:
Not without some refactoring. Here is the current order of operations.
- CreateApplication
- CreateHandler
- OnExecuteRequestHandler
- LoadApplication

I think we need to move LoadApplication before creating the handler. I'll take a quick stab at it and see how hard it is.

Reply:
Will be fairly difficult. Let's not do this in this PR IMO.